bokehjs/src/lib/core/util/date.ts

```diff
@@ -0,0 +1,14 @@
+export function format_date_rfc7231(date: Date): string {
+  const dayNames = ["Sun", "Mon", "Tue", "Wed", "Thu", "Fri", "Sat"]
+  const monthNames = ["Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Sep", "Oct", "Nov", "Dec"]
+
+  const dayOfWeek = dayNames[date.getUTCDay()]
```

Comment:
We use snake case for bokehjs defined names, so `day_of_week`, etc. Sadly currently can't lint this with existing tooling, because they confuse they confuse external library symbols with bokehjs'.